Admin-Component/messages/pt.json

```diff
@@ -1,9 +1,6 @@
 {
```

Comment:
Please add a `TODO:` comment so we don't forget to add he portuguese translation 👌